doc/contributing/releases.md

```diff
@@ -448,6 +448,23 @@ testing is required. Use this opportunity particularly to test macOS and Windows
 installers if there are any concerns. Click through to the individual workers
 for a run to find the artifacts.
 
+Release build list
```

Comment:
I'm not sure about including this list -- it's only accurate for current and not, for example, Node.js 14 or 16.